### PR TITLE
Fix a few bugs in PBRs

### PR DESCRIPTION
**Pull request by flsmith**
1) Display was broken for certain PBRs
2) PBRs weren't required to have sorted adjacencies, which could result in mathematically equal PBRs being nonequal.

### CODE REVIEW

File: src/element.cpp [hunk 1]
```diff
@@ -649,6 +659,14 @@ namespace libsemigroups {
         }
       }
     }
+    for (size_t u = 0; u < n; ++u) {
+      if (!std::is_sorted(this->_vector.at(u).cbegin(),
+                          this->_vector.at(u).cend())) {
+        LIBSEMIGROUPS_EXCEPTION("the adjacencies of vertex ",
+                                detail::to_string(u),
```
Comment (james-d-mitchell) [inline src/element.cpp:666]: This needs to be `detail::to_string(u).c_str()` to work when `fmt` is not enabled.

Comment (james-d-mitchell) [reply src/element.cpp:666]:
```suggestion
                                detail::to_string(u).c_str(),
```

File: src/element.cpp [hunk 2]
```diff
@@ -860,13 +878,17 @@ namespace libsemigroups {
...
-        if (x < 0) {
-          v.push_back(static_cast<uint32_t>(n - x - 1));
-        }
         if (x > 0) {
           v.push_back(static_cast<uint32_t>(x - 1));
         }
       }
+      // We have to go backwards through the vector to add the negative entries,
+      // so that users can input those negative entries in the natural order
+      for (auto it = vec.rbegin(); it < vec.rend(); ++it ) {
```
Comment (flsmith) [inline src/element.cpp:887]:
```suggestion
      for (auto it = vec.rbegin(); it < vec.rend(); ++it) {
```

File: src/element.cpp [hunk 3]
```diff
@@ -881,13 +903,15 @@ namespace libsemigroups {
...
               x);
         }
-        if (x < 0) {
-          v.push_back(static_cast<uint32_t>(n - x - 1));
-        }
         if (x > 0) {
           v.push_back(static_cast<uint32_t>(x - 1));
         }
       }
+      for (auto it = vec.rbegin(); it < vec.rend(); ++it ) {
```
Comment (flsmith) [inline src/element.cpp:910]:
```suggestion
      for (auto it = vec.rbegin(); it < vec.rend(); ++it) {
```

**Pull request merged**